flow/global.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+/* eslint-disable */
+
+// Add missing flow definitions, can be removed when included in flow.
+declare function requestIdleCallback(callback: any): number;
+declare function cancelIdleCallback(idleCallbackId?: any): void;
```

Comment:
Can we make it `idleCallbackId: ?number` instead of `idleCallbackId?: any`?

`idleCallbackId?: any` doesn't make much sense since it implies that you can just call `cancelIdleCallback` without any argument. `idleCallbackId: ?number` is better since you have to pass a number as argument, but you can pass null as well, so no need to check.